### ISSUE TITLE
use_mmap seems to be ignored

### ISSUE THREAD

**Issue by jakedotio** [pyexcel/pyexcel#94]
I'm trying to use pyexcel in a Google App Engine app. The environment does not support mmap.

Using `use_mmap=False` does not seem to work.

```
import pyexcel as pe
video_sheet = pe.get_sheet(file_name=filename, sheet_name='Video', use_mmap=False)
```

Results in an ImportError

```  
File "/code/main/sync.py", line 317, in generate_backup
    video_sheet = pe.get_sheet(file_name=filename, sheet_name='Video', use_mmap=False)
  File "lib/pyexcel/core.py", line 36, in get_sheet
    named_content = sources.get_sheet_stream(**keywords)
  File "lib/pyexcel/internal/core.py", line 20, in get_sheet_stream
    sheets = a_source.get_data()
  File "lib/pyexcel/plugins/sources/file_input.py", line 35, in get_data
    sheets = self.__parser.parse_file(self.__file_name, **self._keywords)
  File "lib/pyexcel/plugins/parsers/excel.py", line 17, in parse_file
    return self._parse_any(file_name, **keywords)
  File "lib/pyexcel/plugins/parsers/excel.py", line 35, in _parse_any
    anything, file_type=file_type, **keywords)
  File "lib/pyexcel_io/io.py", line 65, in get_data
    streaming=False, **keywords)
  File "lib/pyexcel_io/io.py", line 70, in _get_data
    if isstream(afile):
  File "lib/pyexcel_io/_compact.py", line 59, in isstream
    import mmap
  File "/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/python/sandbox.py", line 1039, in load_module
    raise ImportError('No module named %s' % fullname)
ImportError: No module named mmap
```

**Comment by chfw** [comment]
well `use_mmap` was not a flag of pyexcel as far as I remember :)

I will look at the identification google app engine. At the moment, mmap support is limitted to python 2.7+. python 2.6 won't have it. And also will see if `use_mmap` could be a flag otherwise.

**Comment by chfw** [comment]
please help verify the fix:

    pip install https://github.com/pyexcel/pyexcel-io/archive/master.zip

**Comment by jakedotio** [comment]
The fix worked for me. Thank you!

**Comment by chfw** [comment]
released with pyexcel-io v0.4.3. Thanks for your confirmation.
